src/node_debug_options.cc

```diff
@@ -94,18 +94,18 @@ bool DebugOptions::ParseOption(const std::string& option) {
     argument = option.substr(pos + 1);
   }
 
+  // Note that --debug-port and --debug-brk are undocumented, but to be
+  // supported until 7.x is no longer supported, not even in LTS (see #12364).
```

Comment:
As an odd numbered major 7.x will no longer be supported as of June 2017 (in just over a month), so I'm not certain that this comment makes sense.

Also, the issue reference should be a full URL